src/MariaDB/custom/Get-AzMariaDBConnectionString.ps1

```diff
@@ -0,0 +1,138 @@
+
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+function Get-AzMariaDbConnectionString {
+    [OutputType([System.String])]
+    [CmdletBinding(DefaultParameterSetName='ServerName', PositionalBinding=$false)]
+    param(
+        [Parameter(ParameterSetName='ServerName', Mandatory, HelpMessage='The name of the server.')]
+        [Alias('ServerName')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MariaDb.Category('Path')]
+        [System.String]
+        # The name of the server.
+        ${Name},
+    
+        [Parameter(ParameterSetName='ServerName', Mandatory, HelpMessage='The name of the resource group that contains the resource.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MariaDb.Category('Path')]
+        [System.String]
+        # The name of the resource group that contains the resource.
+        # You can obtain this value from the Azure Resource Manager API or the portal.
+        ${ResourceGroupName},
+    
+        [Parameter(ParameterSetName='ServerName', HelpMessage='The subscription ID is part of the URI for every service call')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MariaDb.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MariaDb.Runtime.DefaultInfo(Script='(Get-AzContext).Subscription.Id')]
+        [System.String]
+        # Gets the subscription Id which uniquely identifies the Microsoft Azure subscription.
+        # The subscription ID is part of the URI for every service call.
+        ${SubscriptionId},
+    
+        [Parameter(ParameterSetName='ServerObject', ValueFromPipeline, Mandatory, HelpMessage='Identity Parameter')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MariaDb.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MariaDb.Models.Api20180601Preview.IServer]
+        # Identity Parameter
+        # To construct, see NOTES section for INPUTOBJECT properties and create a hash table.
+        ${InputObject},
+
+        [Parameter(Mandatory, HelpMessage='Connect client type')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MariaDb.Category('Path')]
+        [Validateset('ADO.NET', 'JDBC', 'Node.js', 'PHP', 'Python', 'Ruby', 'WebApp')]
```

Comment:
To use argument may be better here. See https://github.com/Azure/autorest.powershell/blob/master/docs/directives.md#Argument-Completers for details.